### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -15900,6 +15905,7 @@ gerenates->generates
 gerenating->generating
 gerenation->generation
 gerenations->generations
+gerenic->generic
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:15908]:
```text
gerenics->generics
```
https://github.com/search?q=gerenics&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -16552,6 +16558,7 @@ hieght->height
 hiena->hyena
 hierachical->hierarchical
 hierachies->hierarchies
+hierachry->hierarchy
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:16561]:
```text
hierachries->hierarchies
```
https://github.com/search?q=hierachries&type=code

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -18203,6 +18210,7 @@ inposible->impossible
 inpossible->impossible
 inpout->input
 inpouts->inputs
+inpractical->impractical
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:18213]:
```text
inpracticality->impracticality
inpractically->impractically
```
https://github.com/search?q=inpracticality&type=code
https://github.com/search?q=inpractically&type=code

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -21252,6 +21260,7 @@ musel->muscle, mussel,
 musels->muscles, mussels,
 musil->muscle, mussel,
 musils->muscles, mussels,
+musl->must
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:21263]: I suggest you drop this or you move to code. May have significant impact to many.
**musl** is C standard library normally uses in embedded Linux (OpenWRT).
https://musl.libc.org/
https://en.wikipedia.org/wiki/Musl

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -34083,6 +34098,9 @@ unrestrcited->unrestricted
 unrgesiter->unregister
 unroated->unrotated
 unrosponsive->unresponsive
+unrotaete->unrotate
+unrotaeted->unrotated
+unrotaetes->unrotates
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:34103]: I suggest you drop these. Suggested words are misspelling in major & reputable online dictionaries.
They are misspelling in aspell, hunspell, ispell, chrome browser, aoo-mozilla-en-dict, etc.

https://www.google.com/search?q=define+unrotate
https://www.lexico.com/search?filter=en_dictionary&query=unrotate
https://www.ldoceonline.com/spellcheck/english/?q=unrotate
https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=unrotate

https://dictionary.cambridge.org/spellcheck/english/?q=unrotate
https://www.dictionary.com/misspelling?term=unrotate
https://www.merriam-webster.com/dictionary/unrotate

https://www.lexico.com/search?filter=en_dictionary&query=unrotated
https://www.lexico.com/search?filter=en_dictionary&query=unrotates

**Review by vikivivi — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: [9ff34f6](https://github.com/codespell-project/codespell/pull/2430/commits/9ff34f68a714dd26407bd56c9fcbeddd96e911da)

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: [5b1ecbc](https://github.com/codespell-project/codespell/pull/2430/commits/5b1ecbcfd44c3e632f118ab3a72b92691e2b6dd3)

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:18365]: [b8ef1c7](https://github.com/codespell-project/codespell/pull/2430/commits/b8ef1c707ccaae8e76d531a075337ec0682bd7f0)

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:16696]: [b8ef1c7](https://github.com/codespell-project/codespell/pull/2430/commits/b8ef1c707ccaae8e76d531a075337ec0682bd7f0)

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:16041]: [b8ef1c7](https://github.com/codespell-project/codespell/pull/2430/commits/b8ef1c707ccaae8e76d531a075337ec0682bd7f0)

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**